src/discordmessageprocessor.ts

```diff
@@ -79,10 +79,27 @@ export class DiscordMessageProcessor {
     public async FormatEdit(
         oldMsg: Discord.Message,
         newMsg: Discord.Message,
+        link?: string,
     ): Promise<DiscordMessageProcessorResult> {
-        // TODO: Produce a nice, colored diff between the old and new message content
+        // https://matrix.to/#/!room/$event
```

Comment:
what's this for?

Reply:
wooops, old comment soru had in there to remember waht the matrix.to url format looks like XD artifact of development, will remove